src/components/common/modal/folderNameInput/folderNameInput.tsx

```diff
@@ -1,9 +1,9 @@
-import { useState } from 'react';
+import { ChangeEvent, useState } from 'react';

```

Comment:
```suggestion
import type { ChangeEventHandler } from 'react';
import { useState } from 'react';
```

type import 는 이렇게 분리해주시면 값과 타입을 명시적으로 분리할 수 있습니다.